src/components/ImageDetails/ImageDetails.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Input, Box, CloseButton } from '@chakra-ui/react';
+
+function ImageDetails({ index, name, preview, setFilesIntermediate }) {
+    // setFilesIntermediate(prev => {
+    //     prev[i].description = '';
+    //     return prev;
+    // })
+  return (
+    <Box>
+      <p>{name}</p>
+      <CloseButton />
+      <img
+        key={preview}
+        alt="pic"
+        src={preview}
+        // onLoad={() => {
+        //   URL.revokeObjectURL(file.preview);
+        // }}
+      />
+      <Input placeholder="Add description" />
+    </Box>
+  );
+}
+
+ImageDetails.propTypes = {
+  index: PropTypes.number.isRequired,
+  name: PropTypes.isRequired,
+  preview: PropTypes.isRequired,
+  removeImage: PropTypes.func,
```

Comment:
🚫 **[eslint]** <[react/no-unused-prop-types](https://github.com/jsx-eslint/eslint-plugin-react/tree/master/docs/rules/no-unused-prop-types.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'removeImage' PropType is defined but prop is never used